### PR TITLE
Enable recommends for MicroOS GNOME desktop

### PR DESCRIPTION
**Pull request by fcrozat**
ensure language subpackages are installed by default

### CODE REVIEW

File: control/control.MicroOS.xml [hunk 1]
```diff
@@ -263,6 +263,7 @@ textdomain="control"
        </globals>
 	      <software>	
             <default_patterns>microos_base microos_base_packagekit microos_defaults microos_hardware microos_gnome_desktop container_runtime bootloader</default_patterns>
+            <install_recommended config:type="boolean">true</install_recommended>
```
Comment (jreidinger) [inline control/control.MicroOS.xml:266]: well, this element is not known in software. @lslezak what is correct key here?

Comment (lslezak) [reply control/control.MicroOS.xml:266]: ~~Um, that should be correct: https://github.com/yast/yast-autoinstallation/blob/master/src/autoyast-rnc/software.rnc#L21~~

Comment (lslezak) [reply control/control.MicroOS.xml:266]: Ooops, wrong file, it is: https://github.com/yast/yast-installation-control/blob/master/control/control.rnc#L422-L452

Comment (lslezak) [reply control/control.MicroOS.xml:266]: This is not supported, I think the correct solution is to disable the minimal libzypp config, see https://github.com/yast/skelcd-control-MicroOS/blob/948103a55cad761315ebadba3bc027f6f5a048b8/control/control.MicroOS.xml#L118

Comment (lnussel) [reply control/control.MicroOS.xml:266]: also would have to undo the `minimalistic_libzypp_config` hack

Comment (lslezak) [inline control/control.MicroOS.xml:266]: Proposed fix:

```suggestion
            <minimalistic_libzypp_config config:type="boolean">false</minimalistic_libzypp_config>
```

:warning: **Note: Untested!**

Comment (fcrozat) [reply control/control.MicroOS.xml:266]: If you give me an easy way to test an install with this change, I'll be happy to report (and update my PR).

Comment (lnussel) [reply control/control.MicroOS.xml:266]: found this code to create a DUD from my previous control XML experiments https://gist.github.com/lnussel/87f22be05ae8ff2def6d7b9df01f60ce

Comment (fcrozat) [reply control/control.MicroOS.xml:266]: I've tested this as dud but recommends aren't "enabled" if checking in the software section of the installer. Not sure if it is expected.

**Comment by sysrich** [comment]
I really do not like this suggestion.

If we need to install language subpackages by default, lets put the langauge packages in the correct pattern.

I do not want to pull through non-essential packages into MicroOS, which using 'recommended' WILL pull in.

I do want language support though, I consider that essential.

**Comment by fcrozat** [comment]
> I really do not like this suggestion.
> 
> If we need to install language subpackages by default, lets put the langauge packages in the correct pattern.

This means hardcoding in patterns something which is currently dynamic (handled by libzypp) and prone to be incomplete over time (patterns usually list only leaf packages, not all the dependencies). 

> I do not want to pull through non-essential packages into MicroOS, which using 'recommended' WILL pull in.

Understood but I don't see but OTOH, languages subpackages are currently handled as "non essential" packages. 

> I do want language support though, I consider that essential.
+1

**Comment by sysrich** [comment]
> Understood but I don't see but OTOH, languages subpackages are currently handled as "non essential" packages.

By inferior versions of openSUSE maybe

**Comment by fcrozat** [comment]
Let's move the discussion to another place. I'm closing this PR and will follow up on https://bugzilla.opensuse.org/show_bug.cgi?id=1196547

**Pull request closed without merging**